README.md

```diff
@@ -5,8 +5,11 @@ Using the repository template
 
```

Comment:
I think the readme could use some sprucing up. perhaps that's another task where we improve all our docs.